lib/process/process.go

```diff
@@ -1972,6 +1973,11 @@ func (p *Process) initClusterCertificate(ctx context.Context, client *kubernetes
 		return trace.Wrap(err)
 	}
 
+	err = p.replaceCertIfAboutToExpire(ctx, p.client)
```

Comment:
Hmm, if the intention of this is to detect an old expiring certificate upon process start, then shouldn't this be called above? Otherwise this function (`initClusterCertificate`) will detect in the beginning that the certificate is already initialized and exit, thus this code will never really be reached.

Reply:
The intention of this is to handle the case of the cert being deleted using:
```
 gravity resource rm tls keypair
```
In this case the cert will be loaded from the file and this check will verify that the cert from the file hasn't expired. 

I will also add it above to verify the cert on processed restart.